src/osd/SnapMapper.cc

```diff
@@ -240,6 +255,11 @@ void SnapMapper::add_oid(
        ++i) {
     to_add.insert(to_raw(make_pair(*i, oid)));
   }
+  if (g_conf->subsys.should_gather(ceph_subsys_osd, 20)) {
```

Comment:
Aside: I think in the past you've done this exclusion with some macro shenanigans, enclosing the whole loop in a dout/dendl pair. This format here is perfectly (perhaps more) readable, but did you switch for any particular reason?